src/include/daos_srv/evtree.h

```diff
@@ -411,21 +388,8 @@ int evt_create(uint64_t feats, unsigned int order, struct umem_attr *uma,
  * \return		0	Success
  *			-ve	error code
  */
-int evt_create_inplace(uint64_t feats, unsigned int order,
-		       struct umem_attr *uma, struct evt_root *root,
-		       daos_handle_t coh, daos_handle_t *toh);
-/**
- * Open a tree by its memory ID \a root_mmid
- *
- * \param root_mmid	[IN]	Memory ID of the tree root
- * \param uma		[IN]	Memory class attributes
- * \param toh		[OUT]	The returned tree open handle
- *
- * \return		0	Success
- *			-ve	error code
- */
-int evt_open(TMMID(struct evt_root) root_mmid, struct umem_attr *uma,
-	     daos_handle_t *toh);
+int evt_create(uint64_t feats, unsigned int order, struct umem_attr *uma,
```

Comment:
Note that I only removed the evt_open/create and replaced them with inplace versions.   Since that is all we actually use in VOS, I see no point in maintaining the other version